build.cake

```diff
@@ -1,3 +1,10 @@
+// Load the extensions to create the chocolatey package
+#tool "NUnit.Extension.VSProjectLoader"
+#tool "NUnit.Extension.NUnitProjectLoader"
+#tool "NUnit.Extension.NUnitV2ResultWriter"
+#tool "NUnit.Extension.NUnitV2Driver"
+#tool "NUnit.Extension.TeamCityEventListener"
+
```

Comment:
I think I can optimize this a little more and only load them in the task where the chocolatey package is being build.